.buildkite/pipeline.yaml

```diff
@@ -25,7 +25,7 @@ steps:
 
   - label: ":mac: test-static-sanitized.sh (master only)"
```

Comment:
This is no longer `master only`.

I might have also just deleted the line instead of commenting it out, but that's your call.

Reply:
Yes, I made a comment indicating that I was doing this. I'm happy to remove the line.

Reply:
I've removed the line.